account_cash_invoice/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2017 Creu Blanca
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+{
+    'name': 'Account cash invoice',
+    'version': '12.0.0.0.1',
```

Comment:
Should we always use 12.0.1.0.0 for the first version?
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0#tasks-to-do-in-the-migration

Reply:
True, just changed it!